Modules/Microsoft.PowerShell.SecretsManagement/README.md

```diff
@@ -155,7 +157,7 @@ However, it should always implement secret retrieval (GetSecret, GetSecretInfo).
 If a vault extension doesn't support some functionality (such as add/remove a secret, or a particular secret type), then it should throw an exception with a meaningful error message.  
 
 Both of the binary and script modules implement the four functions in similar ways.
-The method/functions take the same parameters, including optional additional parameters, and return the same object(s).
+The method/functions take the same parameters, including optional additional parameters, and return the same object(s).  
```

Comment:
remove trailing spaces (start using vs code and have it ode this for you)

Reply:
Removed.